<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupWidget</class>
 <widget class="QWidget" name="GroupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="ListWidget" name="groupsView">
     <property name="styleSheet">
      <string notr="true">border: 0px solid white</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="buttonFrame">
     <property name="styleSheet">
      <string notr="true">
#buttonFrame {
background-image: url(:icons/backgrounds/291.jpg);
border: none;
padding: 0px;
}

#buttonFrame QPushButton {
color: #333;
border: none;
border-radius: 0px;
padding: 0px;
background: none;
}

#buttonFrame QProgressBar {
color: #222;
border: 2px solid #555;
border-radius: 5px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #433);
min-width: 80px;
}

#buttonFrame QLCDNumber {
color: #222;
border: 2px solid #555;
border-radius: 5px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #433);
min-width: 80px;
}




</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="5">
       <widget class="QProgressBar" name="progressBar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="groupLoadButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconResources.qrc">
          <normaloff>:/icons/icons/1367893718_arrow_bottom.png</normaloff>:/icons/icons/1367893718_arrow_bottom.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="connectButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconResources.qrc">
          <normaloff>:/icons/icons/1367893332_globe_3.png</normaloff>:/icons/icons/1367893332_globe_3.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="showAllButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconResources.qrc">
          <normaloff>:/icons/icons/1367893874_playback_reload.png</normaloff>:/icons/icons/1367893874_playback_reload.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconResources.qrc">
          <normaloff>:/icons/icons/1367893725_zoom.png</normaloff>:/icons/icons/1367893725_zoom.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLCDNumber" name="byteDisplay">
        <property name="numDigits">
         <number>10</number>
        </property>
        <property name="digitCount">
         <number>10</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header>listwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="iconResources.qrc"/>
 </resources>
 <connections/>
</ui>
